### ISSUE TITLE
✅ 🚀 Feature Request: Loop Grid - Include dynamic post classes (post-type, terms, etc.) for each Item in the loop like in the Posts widget

### ISSUE THREAD

**Issue by wwdes** [elementor/elementor#20213]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently css is unable to target loop items based on post type. Some loop queries may combine different post types, which is where my concern comes from as minor css adjustments can be made for different types of content that a loop may show.

### Describe the solution you'd like

Add some more data to loop item elements. Alternatively, setup filters so that we can add attributes that we need our selves.

**Comment by nicholaszein** [comment]
Hello @wwdes,

thanks for submitting your feature request! 🙏 

Please understand that while we do welcome new feature requests, we can't promise they will be considered or added to our roadmap for the time being. Allow us to take some time to review your request. We will get back to you with feedback as soon as possible.

Best regards

**Comment by wwdes** [comment]
Understandable.

I may just add that the classic Posts widget has this feature already and hopefully will make in to Loop Grid widget too.

**Comment by nicholaszein** [comment]
Hey @wwdes,

I just wanted to let you know that I've labeled your issues so our team can find them in the future when considering new features to add in the next phases.

Alternatively, you could try to create a shortcode that fetches the current queried object and output its terms and post-type slugs. This way you could use the _Dynamic-tag > Shortcode_ in the main container's CSS classes, achieving what you want for now.

I hope this helps guide you in the right direction.

Best regards,

Nick

**Comment by huubl** [comment]
Maybe this will help someone. I use this hook to add classes to the loop item:

```
function loop_wrapper_attributes( $attributes, $document ) {
	if ( 'loop-item' === $document::get_type() ) {
		$categories = get_the_category();
		foreach( $categories as $category ) {
			$attributes['class'] .= ' category-'.$category->slug;
		}
	}

	return $attributes;
}
add_filter( 'elementor/document/wrapper_attributes', 'loop_wrapper_attributes', 2, 20 );
```

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.12**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
